client/src/components/SingleVideo.css

```diff
@@ -0,0 +1,55 @@
+.video-card {
+  text-align: center;
+  box-shadow: 1px 4px 18px -1px rgba(0, 0, 0, 0.4);
+  padding: 1rem;
+}
+
+.votes-container {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  /* background-color: aquamarine; */
```

Comment:
It's good to get into the habit of removing commented out code that you're not using - if you commit your work regularly you can _always_ go back to what you had before, so don't be scared to take stuff out.